platform/linux-generic/odp_random_null.c

```diff
@@ -29,7 +34,7 @@ static int32_t _random_data(uint8_t *buf, uint32_t len, uint32_t *seed)
 	while (i < len) {
 		u.rand_word = rand_r(seed);
 
-		for (j = 0; j < 4 && i < len; j++, i++)
+		for (j = 0; j < NUM_RAND_BYTES && i < len; j++, i++)
```

Comment:
This does not work correctly with big endian byte order as it would use the most significant bits returned by rand_r(). I would suggest either changing the code to endian-neutal or adding an assert about byte order.

Reply:
Fixed in V2.